src/tests/whitespace_test

```diff
@@ -35,3 +35,15 @@ fi
             exit found
         }
     "
+
+declare found_file=0
+git ls-files | \
+    grep -v "^src/config/testconfigs/noparse.api.conf" | \
+    grep -v "^src/tests/cmocka/p11_nssdb/.*db" | \
+    while read file; do
+        test `tail -c 1 $file` && \
```

Comment:
Could you perhaps write `test -n ...` instead. Although, probably fine as it is, it makes me feel uncomfortable about the reliability of the construct :) Plus this way it is a bit clearer what's being tested for.